src/test/webapp/js/specs/nearyouServiceTest.js

```diff
@@ -0,0 +1,55 @@
+describe('the nearyou service', function() {
+  var nearyouService;
+
+  beforeEach(function() {
+    nearyouService = new nearyouScript();
+    events = [];
+    eventObjs = [];
+    currLocation = new Object();
+    currLocation.xlocation = 0;
+    currLocation.ylocation = 0;
+  });
+
+  it('must detect when there are no events', function () {
```

Comment:
Nit: By convention, test cases should be phrased as statements of fact about the behavior of the system under test, e.g., `it("detects when there are no events")`

Reply:
Thanks for that, I did not know that